src/lib/salad/bloom.h

```diff
@@ -68,57 +68,60 @@ struct bloom_block {
 };
 
 /**
- * Bloom filter data structure
+ * Bloom filter header, data is stored separately: all the methods accept
```

Comment:
I refer to it as description or something. Header sounds like we dealing with serialization or file structure.